gds/vector/cosine_distance.gsql

```diff
@@ -0,0 +1,56 @@
+CREATE FUNCTION gds.vector.cosine_distance(list<double> list1, list<double> list2) RETURNS(float) {
```

Comment:
what does "/gds" mean here?

Reply:
I think it should be short for `graph database system`, Neo4j used this as their built-in library.

Reply:
GDS means Graph Data Science.

Reply:
Would returning double be better since float has lower precision? Same applies below.

Reply:
It will introduce some type checking error since they are mainly used for float types.